app/javascript/components/Ramp.scss

```diff
@@ -28,16 +28,62 @@
   }
 }
 
+.ramp--rails-title {
+  font-size: 2rem !important;
+  margin-top: 1rem !important;
+}
+
 .ramp--rails-content {
   display: flex;
 
+  #administrative_options {
+    text-align: right !important;
+  }
+
+  .btn-sm {
+    padding: 0 0;
+  }
+
   .share-tabs {
     flex-grow: 2;
   }
-  
+
   #share-list {
     margin-left: -9.25rem;
   }
+
+  .sr-only {
+    border: 0;
+    clip: rect(0 0 0 0);
+    height: 1px;
+    margin: -1px;
+    overflow: hidden;
+    padding: 0;
+    position: absolute;
+    width: 1px;
+  }
```

Comment:
`.sr-only` is a Bootstrap class. I'm not sure whether we need add this CSS in the Avalon's code. This should be already coming in from the Bootstrap's CSS via the asset pipeline. I think we could remove this from this file?